dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -916,6 +918,22 @@ public Workflow manualRegister(@ApiParam(hidden = true) @Auth User user,
             @ApiParam(value = "Descriptor type", required = true) @QueryParam("descriptorType") String descriptorType,
             @ApiParam(value = "Default test parameter file path") @QueryParam("defaultTestParameterFilePath") String defaultTestParameterFilePath) {
 
+        // Set up source code interface and ensure token is set up
+        // construct git url like git@github.com:ga4gh/dockstore-ui.git
+        String registryURLPrefix;
+        SourceControl sourceControlEnum;
+        if (workflowRegistry.toLowerCase().equals("bitbucket")) {
```

Comment:
I don't think these values are defined anywhere, let's add them to the (swagger) description of workflowRegistry above